test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,25 @@
+require "test_helper"
+
+describe UsersController do
+  describe "login" do
+    it "can log in a user" do
+    
```

Comment:
There are 3 different behaviors I would be interested in testing here:
- Log in a new user
- Log in a returning user
- Fail to log in (e.g. send "" as the username)